imod_coupler/drivers/dfm_metamod/mf6_utilities.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from imod import mf6
```

Comment:
You cannot depend on imod-python :)

Reply:
(in the main code at least, the tests are fine)

Reply:
done. removed these imports. They were unused.

Reply:
Nice, now the testbench is green too